sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -895,10 +896,12 @@ class Analyzer(
   object ResolveTables extends Rule[LogicalPlan] {
     def apply(plan: LogicalPlan): LogicalPlan = ResolveTempViews(plan).resolveOperatorsUp {
       case u: UnresolvedRelation =>
-        lookupV2Relation(u.multipartIdentifier, u.options)
-          .map { rel =>
-            val ident = rel.identifier.get
-            SubqueryAlias(rel.catalog.get.name +: ident.namespace :+ ident.name, rel)
+        lookupV2Relation(u.multipartIdentifier, u.options, u.isStreaming)
+          .map {
+            case rel: DataSourceV2Relation =>
+              val ident = rel.identifier.get
+              SubqueryAlias(rel.catalog.get.name +: ident.namespace :+ ident.name, rel)
```

Comment:
shouldn't we add `SubqueryAlias` to streaming relation as well?

Reply:
Sure, done in fad1976.